### PR TITLE
Mode byrummnet only has wms maps

### PR DESCRIPTION
**Pull request by ITViking**
This is the finalizing PR on making the new maps work in the mode byrummet. 

image

# Changes
We can now select between the 4 (test) maps we have from GEO. 
All functionality should still be there. The maps should only be available to Byrummet for now. 

## Known Issues
The load and render of the WMS pictures are really slow (GEO server fault - it takes upwards of 16 seconds to deliver). Because of the TimeWarp implementation, the pink circle doesn't show up until the map is ready. That leaves the user without any really good indicator of something happening. 
I don't have a fix on my hands. 
image

## How to test
For every map, test that the map can be interacted with. Assets chosen, mouse used to drag, zoom in and out. Use the arrow keys to make sure that they can still be used to move the map around. 
Go to a single assets page, go the map, switch between the maps and ensure that no new maps have become available. 
Go to map search and check for the same thing. 
Now go to **Byrummet**. There should be no more than 4 maps to chose from.

### CODE REVIEW

File: assets-pipeline/app/scripts-browserify/historisk-atlas/prepareTimeWarp.js [hunk 1]
```diff
@@ -22,16 +22,33 @@
...
+  if(submode && submode === 'byrummet') {
+    //If we are in mode 'byrummet' we only have WMS maps so we only create a layer for that
+    timeWarp = new ol.layer.Image({
+      source: new ol.source.ImageWMS({
+        crossOrigin: "*",
+        hidpi: false,
+        params: {
+          LAYERS: mapId
+        },
+        url: 'https://data.geo.dk/mapv2/geo-services/wms?TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1bmlxdWVfbmFtZSI6Imdlb2F0bGFzQEJVUi5kayIsIkdBTC5Jc0F1dGhlbnRpY2F0ZWQiOiIxIiwiR0FMLlVzZXJOYW1lIjoiZ2VvYXRsYXNAQlVSLmRrIiwiR0FMLk9yZ2FuaXNhdGlvbiI6IiIsIkdBTC5FbWFpbCI6Imdlb2F0bGFzQGJ1ci5kayIsIkdBTC5HZW9GdW5jdGlvbnMiOiIqIiwiR0FMLkdlb0RhdGFTb3VyY2VzIjoiIiwiR0FMLkdlb01hcHMiOiJCVVIiLCJHQUwuR2VvTW9kZWxzIjoiLTEiLCJHQUwuSHlkcm9Nb2RlbHMiOiIiLCJHQUwuR2VvVGVtcGxhdGVzIjoiIiwiR0FMLkdWTExvZ2dlcnMiOiIiLCJHQUwuR2VvQXJlYSI6IjEiLCJHQUwuQmF0aHltZXRyeSI6IiIsIkdBTC5Cb3JlaG9sZUxpbmsiOiIiLCJHQUwuUHJvZmlsZUxheWVycyI6IiIsIkdBTC5Qcm9qZWN0IjoiIiwiR0FMLlByb2plY3RDb2RlIjoiIiwiR0FMLlByb2plY3RBZG1pbiI6IiIsIkdBTC5HZW9NYXBMYXllclR5cGVzIjoiMjIiLCJHQUwuRW52U2FtcGxlU291cmNlcyI6IiIsIm5iZiI6MTY5NjYwMTUzNCwiZXhwIjoxODU0MjgxNTM0LCJpYXQiOjE2OTY2MDE1MzR9.jf2J1xzX3z9SHI2nY7euV1DaEeEoyqnL4lnow9O2AyU'
```
Comment (Arth101) [inline assets-pipeline/app/scripts-browserify/historisk-atlas/prepareTimeWarp.js:40]: Synes vi skal trække `TOKEN` ud i config. Så er Url'en også nemmere at læse.

File: assets-pipeline/app/scripts-browserify/historisk-atlas/prepareMap.js [hunk 2]
```diff
@@ -21,9 +21,13 @@ module.exports = function prepareMap(mapElement, center, offset, zoomLevel, time
 
     // When select is changes update the source of the appropriate source.
     mapState.mapSelectElement.addEventListener('change', function (event) {
-      var source = mapState.isSearchMode() ? mapState.timeWarp.getSource() : mapState.rasterLayer.getSource();
       mapState.id = mapState.mapSelectElement.value;
-      source.setUrl(mapState.getMapUrl(mapState.id));
+      let source = mapState.isSearchMode() ? mapState.timeWarp.getSource() : mapState.rasterLayer.getSource();
```
Comment (Arth101) [inline assets-pipeline/app/scripts-browserify/historisk-atlas/prepareMap.js:25]: Burde `source` ikke være en `const` eller bliver en overskrevet et andet sted?

File: shared/config/env/base.js [hunk 3]
```diff
@@ -128,16 +128,19 @@ let config = {
...
+      { id: 38, title: '4 cm kort', year: '1953-1976', type: 'tile' },
+      { id: 228, title: 'K\u00F8benhavn', year: '1860', type: 'tile' },
+      { id: 105, title: 'K\u00F8benhavn', year: '1939', type: 'tile' },
+      { id: 54, title: 'Lavkantkort', year: '1901-1945', type: 'tile' },
+      { id: 55, title: 'H\u00F8jkantkort', year: '1840-1899', type: 'tile' },
+      { id: 154, title: 'Geddes kort', year: '1761', type: 'tile' },
+      { id: 'timestep1', title: 'GEO kort', year: '1100', type: 'wms' },
+      { id: 'timestep3', title: 'GEO kort', year: '1200', type: 'wms' },
+      { id: 'timestep4', title: 'GEO kort', year: '1600', type: 'wms' },
+      { id: 'timestep5', title: 'GEO kort', year: '1700', type: 'wms' }
```
Comment (Arth101) [inline shared/config/env/base.js:143]: Har vi nogen info om hvilke kort som høre til hvilken periode?

Kan se GEO siger at de laver stenalder, middelalder, 1600 tallet og 1700 tallet. Vi må få Signe til at sige hvilke kort der er hvilken periode, så kan vi også aktiverer det når man skifter periode i tidslinjen.

**Comment by Arth101** [comment]
Mini-map skal rettes til så det virker med den nye kode før vi kan merge.

File: assets-pipeline/app/scripts-browserify/historisk-atlas/prepareTimeWarp.js (hunk 1 above)
Comment (ITViking) [reply assets-pipeline/app/scripts-browserify/historisk-atlas/prepareTimeWarp.js:40]: det lyder som en god ide

File: assets-pipeline/app/scripts-browserify/historisk-atlas/prepareMap.js (hunk 2 above)
Comment (ITViking) [reply assets-pipeline/app/scripts-browserify/historisk-atlas/prepareMap.js]: fixed

File: assets-pipeline/app/scripts-browserify/historisk-atlas/prepareTimeWarp.js (hunk 1 above)
Comment (ITViking) [reply assets-pipeline/app/scripts-browserify/historisk-atlas/prepareTimeWarp.js]: done

File: shared/config/env/base.js (hunk 3 above)
Comment (ITViking) [reply shared/config/env/base.js]: jeg har forsøgt noget for nu og så må Signe lige melde tilbage

**Comment by ITViking** [comment]
Mini-map fixed

**Pull request merged**